### ISSUE TITLE
Alter P2K16 to have door circles, similar to tools circles. 

### ISSUE THREAD

**Issue by eliasbakken** [bitraf/infrastructure#149]
As a part of the moving to BV9 in September 2021, it would be good to have the possibility to add new door circles that can send messages on a specific topic similar to tools. 
That way people can set up door locks on office doors in the old and new building.

**Comment by omega** [comment]
So we want an admin interface similar to tools for the buttons on the front page?

**Comment by eliasbakken** [comment]
I haven't thought it through yet, maybe it requires a meeting. But i guess
first would be to have support for several door circles, second would be to
show buttons for all circles the user is a member of on the "open" page.

**Comment by trygvis** [comment]
The doors are defined in the code, so the required circle can just be a part of the code.

If we ever make the doors list a part of the database something tools like would be useful.

**Comment by omega** [comment]
@trygvis I thought that was the request here, to make it easier to add more doors..?

**Comment by omega** [comment]
@eliasbakken I guess one thing that needs to be thought about is who can add/administer circles for doors? Like if I set up a new lock on my office, we need to prevent someone else from just adding a new door with the same mqtt-topic etc?

**Comment by trygvis** [comment]
> @trygvis I thought that was the request here, to make it easier to add more doors..?

Nope, just have circle-based authorization in addition to the existing membership/office checks.
